requirements-base.txt

```diff
@@ -109,3 +109,5 @@ phabricator>=0.7.0
 selenium>=4.1.5
 
 openai==0.27.0
+
+litellm==0.1.213
```

Comment:
this pulls in a bunch of dependencies that we don't use

Reply:
is there somewhere else I should add this ? @asottile-sentry 

If you're not trying to add multiple llm providers i'm assuming this is not useful

Reply:
I don't believe we're trying to add multiple llm providers